Libro.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once "Autor.php";
```

Comment:
```suggestion

require_once "Autor.php";
```
para que no sea confuso seria bueno que haya una distancia de un cuadro